_layouts/base_slides.html

```diff
@@ -22,12 +22,12 @@
 class: center, middle, inverse
 
 <div class="my-header"><span>
-<a href="{{ site.baseurl }}/topics/{{ topic.name }}" title="Return to topic page" ><i class="fa fa-level-up" aria-hidden="true"></i></a>
+<a href="{{ site.baseurl }}/topics{{ topic.name }}" title="Return to topic page" ><i class="fa fa-level-up" aria-hidden="true"></i></a>
 </span></div>
 
 <div class="my-footer"><span>
 {% if page.logo == "GTN" %}
-<img src="{{ site.baseurl }}/{{ site.small_logo }}" alt="Galaxy Training Network" style="height: 40px;"/>
+<img src="{{ site.baseurl }}/topics{{ site.small_logo }}" alt="Galaxy Training Network" style="height: 40px;"/>
```

Comment:
@erasche @bagnacan This broke the logo on slides, see e.g. http://galaxyproject.github.io/training-material/topics/dev/tutorials/tool-integration/slides.htm

It's broken also when I run `make serve` locally.

Reply:
Will fix.

Reply:
@nsoranzo @erasche thank you, I missed that.